isy-bedienkonzept-doc/modules/ROOT/pages/bedienkonzept.adoc

```diff
@@ -13,7 +13,7 @@ Es soll sicherstellen, dass Nutzende das Frontend ohne Schwierigkeiten nutzen k
 
 In den folgenden Kapiteln werden die benötigten Elemente zum Erreichen einer herausragenden Benutzeroberfläche von "klein nach groß" beschrieben.
 *Der allgemeine Rahmen und die übergreifenden Prinzipien* werden in den ersten Kapiteln beschrieben.
-Darauf folgen die *<<basiselemente>>* (wie z.B. Text und Farben), die den Grundstein für die dahinterliegenden Kapitel schaffen und essenziell wichtig sind.
+Darauf folgen die *<<basiselemente>>* (wie z.B. <<text>> und <<farben>>), die den Grundstein für die dahinterliegenden Kapitel schaffen und essenziell wichtig sind.
 Danach kommen die *<<bedienelemente>>*, die einfachste Ausprägung eines Interaktionselements.
 *<<design-pattern>>* stellen eine Menge aus Bedienelementen und Basiselementen dar und können letztendlich in beliebiger weiterer Kombination als vollständiger *<<use-cases,Use Case>>* beschrieben werden.
```

Comment:
*<<design-pattern>>* stellen eine Menge aus Bedienelementen und Basiselementen dar und können letztendlich in beliebiger Kombination als vollständige *<<use-cases,Use Cases>>* beschrieben werden.

Reply:
Done